plat/common/arm/traps.c

```diff
@@ -32,6 +36,121 @@ static const char *exception_modes[]= {
 	"Error"
 };
 
+enum aarch64_trap {
```

Comment:
`trap.c` is a generic file. If we add Armv7 support it will bloat. It would be better if we declared these to separate per-arhictecture files.

Reply:
Yes, good point. I renamed the file to `traps_arm64.c` for now. I think, we may want to move all the 64-bit platform files into a `arm64` subdirectory and remove the `_arm64` suffix at some point. Instead we should have a separate `arm32` or so folder for the 32-bit stuff. All common code could remain where it is.